assets/shared/structure/mock-storage.js

```diff
@@ -0,0 +1,45 @@
+let nextId = 1;
```

Comment:
I think we aren't using this file for now right? Do we have future plans for this?

Reply:
Removed in 1eff720